content/docs/legacy/_index.md

```diff
@@ -1,5 +1,8 @@
 +++
 title = "Legacy docs"
-page_template = "legacy-docs.html"
-render = false
+page_template = "docs/legacy.html"
+weight = 900
+[extra]
+emoji = "🕸️"
+tile = "Legacy documentation"
```

Comment:
Should we just use the title instead of a different label? If anything I would expect the tile to be more space-constrained and thus use the "docs" wording instead of the other way around.

Reply:
This works well of the legacy tile but not for the rest of them. I'm not sure it's worth introducing a special case in the template just for the legacy tile, what do you think?

This was introduced to get a proper sentence on the tile ("I want to...") and keep a short title on the documentation page itself